test/specs/pushAndPull.e2e.ts

```diff
@@ -190,14 +165,8 @@ describe('Push and Pull', async () => {
     expect(successNotificationWasFound).toBe(true);
 
     // Check the output.
-    outputPanelText = await utilities.attemptToFindOutputPanelText(
-      'Salesforce CLI',
-      '=== Pushed Source',
-      10
-    );
-    expect(outputPanelText).not.toBeUndefined();
+    const outputPanelText = await utilities.verifyPushAndPullOutputText('Push', 'Changed');
     expect(outputPanelText).toContain('ended with exit code 0');
```

Comment:
See my other comment in miscellaneous.ts - this line can be removed because the helper function would check for it.